rust-connectors/sinks/kafka/Cargo.toml

```diff
@@ -17,5 +17,7 @@ serde = "1"
 tokio-stream = "0.1"
 fluvio-future = { version = "0.4.1", features = ["subscriber"] }
 
-rdkafka = { version = "0.28", features = ["cmake-build"] }
-kafka = "0.9.0"
+# There is some difference between rev v0.26.0 and the 0.26.0 on crates.io.
+# Changing it to use the crates.io version breaks the musl cross compile.
+rdkafka = { git="https://github.com/fede1024/rust-rdkafka", rev="v0.26.0", features = ["cmake-build", "libz-static", "tokio", "ssl-vendored" , "gssapi-vendored"], default-features = false  }
```

Comment:
is this still required?

Reply:
Yeah. This is still required. https://github.com/fede1024/rust-rdkafka/issues/446#issuecomment-1168043359 it's not ideal but has to do with the openssl runtime.